command/common.go

```diff
@@ -0,0 +1,47 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/cli/cli/internal/ghrepo"
+	"github.com/cli/cli/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func getTitle(cmd *cobra.Command, cmdType string, matchCount int, totalMatchCount int, baseRepo ghrepo.Interface) string {
```

Comment:
Thanks for adding to the test suite!

I'd like to see more test coverage for this function, though. Right now the tests only cover the output of this function when there are no results or when there is no filtering.

Reply:
Updated. Also fixed a small bug in resetting command in tests (the limit was not resetting to 30). Could you please take a look now?

Reply:
thanks!